.github/workflows/on_release_publish.yml

```diff
@@ -39,6 +39,18 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.TOKEN_BOT }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      # Chromatic
+      - name: Build Storybook
+        run: yarn build-storybook
```

Comment:
Can't we pass `buildScriptName` to the chromatic action?

> ```buildScriptName: 'The npm script that builds your Storybook [build-storybook]'```

Reply:
Why do you think we need this? 🤔

Reply:
We don't _need_ it. But from the docs I understand that we could pass `buildScriptName: build-storybook` to the chromatic action and it will take care of building storybook, no? And we could remove this previous step ☝️

Reply:
I updated the yml removing the unnecessary steps. We don't need the `buildScriptName` because our build command is the default one (`build-storybook`). 🙏🏻 

Can you check it again, please?